docs/getting_started.md

```diff
@@ -24,19 +23,18 @@ make
 
 The `thanos` binary should now be in your `$PATH` and is the only thing required to deploy any of its components.
 
-## Sidecars
+## [Sidecar](components/sidecar.md)
 
-Thanos integrates with existing Prometheus servers through a sidecar process, which runs on same machine/in the same pod as the Prometheus server itself. It only works with Prometheus instances of version 2.0.
+Thanos integrates with existing Prometheus (v2.2.1+) servers through a [Sidecar process](https://docs.microsoft.com/en-us/azure/architecture/patterns/sidecar#solution), which runs on the same machine or in the same pod as the Prometheus server/s. 
```

Comment:
`/s. ` is not needed? sidecar will run in the same pod as Prometheus server - no way we can have more than 1 Prom container in pod... and that `/s` suggests isn't it?

Reply:
True yeah. I'll change it